test/unit/BondingManager.js

```diff
@@ -132,14 +133,15 @@ contract("BondingManager", accounts => {
 
             describe("transcoder pool is not full", () => {
                 it("should add new transcoder to the pool", async () => {
-                    bondingManager.TranscoderUpdate({transcoder: accounts[0]}).on("data", e => {
-                        assert.equal(e.returnValues.pendingRewardCut, 5, "should fire TranscoderUpdate event with provided rewardCut")
-                        assert.equal(e.returnValues.pendingFeeShare, 10, "should fire TranscoderUpdate event with provided feeShare")
-                        assert.equal(e.returnValues.args.registered, true, "should fire TranscoderUpdate event with registered set to true")
-                    })
-
                     await bondingManager.bond(1000, accounts[0])
-                    await bondingManager.transcoder(5, 10)
+                    const txRes = await bondingManager.transcoder(5, 10)
+
+                    truffleAssert.eventEmitted(
+                        txRes,
+                        "TranscoderUpdate",
+                        e => e.transcoder == accounts[0] && e.rewardCut == 5 && e.feeShare == 10 && e.registered == true,
```

Comment:
Can omit the `true` and just do `e.registered` and if we're doing the opposite check can omit the `false` and just do `!e.registered`. Same comment applies to the rest of the event assertions that include `e.registered`.

Reply:
fixed in ad3cb40